cpp/include/tensorrt_llm/batch_manager/llmRequest.h

```diff
@@ -1772,6 +1780,9 @@ class GenericLlmRequest
 
     LlmRequestState mState{LlmRequestState::kCONTEXT_INIT};
 
+    // Position of current ptable index
```

Comment:
outside prompttuningbuffer context, better call it prompt tuning table so people can understand. ptable in variable name is fine to be simple
is it better to be "// current position of the prompt tuning table"